packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -661,39 +765,175 @@ abstract class TableBase extends Resource implements ITable {
   /**
    * Metric for the system errors this table
    *
-   * @default sum over a minute
+   * @deprecated use `metricSystemErrorsForOperations`.
    */
   public metricSystemErrors(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+
+    if (!props?.dimensions?.Operation || !props?.dimensions?.TableName) {
```

Comment:
Just to ensure I understand the perspective here. A user who used these methods without specifying both of the above would never have a valid metric. So while the synth/deploy would succeed, it would be with a broken metric/alarm/dashboard/etc. The mentality is that this is broken -- despite the successful synth -- and so it's now acceptable for a user's synth to fail to warn them of this fact. Correct?

Reply:
Yeap